integration-cli/docker_cli_daemon_test.go

```diff
@@ -106,12 +106,9 @@ func (s *DockerDaemonSuite) TestDaemonRestartWithVolumesRefs(c *testing.T) {
 		c.Fatal(err, out)
 	}
 
-	out, err := s.d.Cmd("inspect", "-f", "{{json .Mounts}}", "volrestarttest1")
+	out, err := s.d.Cmd("inspect", "-f", `{{range .Mounts}}{{if eq .Destination "/foo"}}FOUND{{end}}{{end}}`, "volrestarttest1")
 	assert.NilError(c, err, out)
-
-	if _, err := inspectMountPointJSON(out, "/foo"); err != nil {
-		c.Fatalf("Expected volume to exist: /foo, error: %v\n", err)
-	}
+	assert.Equal(c, out, "FOUND")
```

Comment:
Pretty sure this'll fix the test case.
```suggestion
	assert.Equal(c, strings.TrimSpace(out), "FOUND")
```

Reply:
oh! I saw it fail earlier, and was wondering if it was a flake or an actual issue; let me fix that one